dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventChangeLogService.java

```diff
@@ -148,34 +148,27 @@ private <T> void logIfChanged(
 
       EventChangeLog eventChangeLog =
           new EventChangeLog(
-              event,
-              null,
-              propertyName,
-              currentValue,
-              newValue,
-              changeLogType,
-              new Date(),
-              userName);
+              event, null, fieldName, currentValue, newValue, changeLogType, new Date(), userName);
 
       hibernateEventChangeLogStore.addEventChangeLog(eventChangeLog);
     }
   }
 
   private ChangeLogType getChangeLogType(String oldValue, String newValue) {
-    if (isNewProperty(oldValue, newValue)) {
+    if (isNewField(oldValue, newValue)) {
       return ChangeLogType.CREATE;
-    } else if (isUpdateProperty(oldValue, newValue)) {
+    } else if (isUpdateField(oldValue, newValue)) {
       return ChangeLogType.UPDATE;
     } else {
       return ChangeLogType.DELETE;
     }
   }
 
-  private boolean isNewProperty(String originalValue, String payloadValue) {
+  private boolean isNewField(String originalValue, String payloadValue) {
```

Comment:
Thoughts about

isFieldCreated()
OR
isFieldNew()

Reply:
I don't think it even needs `field`. This is about whether this is a create, update or delete which we determine based on the values. How about `isCreate`, `isUpdate`, `isDelete`